### PR TITLE
Create page in AMPLIFY Central section “manage-environments-ownership”

### PR DESCRIPTION
**Pull request by georgiana-caras**
Automatically generated by Netlify CMS

### CODE REVIEW

**Comment by georgiana-caras** [comment]
Hi @lbadenhop, can you add this page under this section: https://docs.axway.com/bundle/axway-open-docs/page/docs/central/env_gw_mgmt/index.html?

File: content/en/docs/central/manage-environments-ownership.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
+title: Assign an environment owner
```
Comment (ssmogos) [inline content/en/docs/central/manage-environments-ownership.md:2]: Environment has the first letter in uppercase in some places, but in some is lower case. Can this be changed to be consistent everywhere?  
Examples: "Learn how to limit access to Environments and API Services" vs "limit access to API Services in that environments".

File: content/en/docs/central/manage-environments-ownership.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+description: Learn how to limit access to Environments and API Services by
+  assigning an owning team.
+---
+Within Amplify Central you can configure an environment's ownership to limit access to API Services in that environments. By assigning an owner, Administrators can delegate full control over the API services within that environment to the members of the owning team, as further indicated by their user role. Members outside of the owning team will not be able to view or create API Services in that environment.
```
Comment (ssmogos) [inline content/en/docs/central/manage-environments-ownership.md:6]: First sentence does not sound that good to me :). Can that be rephrased?

**Comment by lbadenhop** [comment]
@robertengaxway Waiting on your review before I can review and merge. Thanks!

File: content/en/docs/central/manage-environments-ownership.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
+---
+title: Assign an environment owner
+description: Learn how to limit access to Environments and API Services by
+  assigning an owning team.
+---
+Within Amplify Central you can configure the environment ownership to limit access to API Services in that environment. By assigning an owner, Administrators can delegate full control over the API services within that environment to the members of the owning team, as further indicated by their user role. Members outside of the owning team will not be able to view or create API Services or any other resources in that scope.
+
+#### Who can configure the environment ownership?
+
+Users that are assigned the Platform Administrator role in combination with Central Administrator role can assign an owning team, either when creating the environment or after creation.
```
Comment (todanator) [inline content/en/docs/central/manage-environments-ownership.md:10]:
```suggestion
Users that are assigned the Platform Administrator role in combination with the Central Administrator role can assign an owning team, either when creating the environment or after creation.
```

File: content/en/docs/central/manage-environments-ownership.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+#### Environment ownership
+
+Setting up an owner for an environment is OPTIONAL. You can create one without configuring the ownership. That means only Central Administrators can view and manage that environment and the API Services within that scope. No other members in the organization will be able to view or access those resources, until an owner is assigned.
```
Comment (todanator) [inline content/en/docs/central/manage-environments-ownership.md:14]:
```suggestion
Environments can be created without configuring the ownership so setting an owner is OPTIONAL. That means only Central Administrators can view and manage that environment and the API Services within that scope. No other members in the organization will be able to view or access those resources, until an owner is assigned.
```

**Review by todanator — COMMENTED** [review]
Just to minor suggestions

**Review by todanator — APPROVED** [review]

File: content/en/docs/central/manage-environments-ownership.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
+---
+title: Assign an environment owner
+description: Learn how to limit access to Environments and API Services by
+  assigning an owning team.
+---
+Within Amplify Central you can configure the environment ownership to limit access to API Services in that environment. By assigning an owner, Administrators can delegate full control over the API services within that environment to the members of the owning team, as further indicated by their user role. Members outside of the owning team will not be able to view or create API Services or any other resources in that scope.
+
+#### Who can configure the environment ownership?
+
+Users that are assigned the Platform Administrator role in combination with the Central Administrator role can assign an owning team, either when creating the environment or after creation.
```
Comment (robertengaxway) [inline content/en/docs/central/manage-environments-ownership.md:10]: "assign an owning team when creating the environment or change the owning team after creation"

File: content/en/docs/central/manage-environments-ownership.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+
+#### Environment ownership
+
+Environments can be created without configuring the ownership so setting an owner is OPTIONAL. That means only Central Administrators can view and manage that environment and the API Services within that scope. No other members in the organization will be able to view or access those resources, until an owner is assigned.
+
+#### API Service ownership
+
+When creating an API Service, you can assign an owner as well. However, the API Service must have the same owner as the environment the service was registered under. If no owner is assigned, then that API Service will inherit the environment's owner. The same rules apply for the rest of the resources.
```
Comment (robertengaxway) [inline content/en/docs/central/manage-environments-ownership.md:18]: "The same rules apply to all the resources created within the environment"

File: content/en/docs/central/manage-environments-ownership.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+  "spec": {
+    "description": "spec description one"
+  }
+ }
+```
+
+When configuring the owner, you need to specify the following properties:
+
+* **type**: the type of the owner. It defaults to "team", if not present. Currently, only the \`team\` type is supported.
+* **id:** the id of the owner. When the type is set to \`team\`, set the value to the id of the team you want to assign as owner.
```
Comment (robertengaxway) [inline content/en/docs/central/manage-environments-ownership.md:54]: Should we include some text around "if the id is not specified or set to double quotes in the json file, the ownership will be assigned to the environment owner where the API Service is created"

**Review by robertengaxway — COMMENTED** [review]
I added some suggested edits.   Do we want to document or mention Sharing access with other teams or is that a separate section?

File: content/en/docs/central/manage-environments-ownership.md (hunk 7 above)
Comment (georgiana-caras) [reply content/en/docs/central/manage-environments-ownership.md:54]: That example is for an Environment. I can add one for assigning an owner to an API Service and add the text that you suggested.

**Comment by lbadenhop** [comment]
@georgiana-caras Please let me know when you are done making changes so I can edit and merge.

**Review by robertengaxway — APPROVED** [review]
Please look at my suggested edits but I approve.

**Comment by georgiana-caras** [comment]
@lbadenhop, I am done making changes.

**Review by lbadenhop — APPROVED** [review]

**Pull request merged**